issue-scan

```diff
@@ -125,6 +127,9 @@ def tasks_for_issues(issues_data):
         if 'bot' in labels:
             issues.append(issue)
         api = github.GitHub(repo=repo)
+    elif opts_repo:
+        api = github.GitHub(repo=opts_repo)
+        issues = api.issues(state="open")
     else:
         api = github.GitHub()
         issues = api.issues(state="open")
```

Comment:
Please DRY. if `opts_repo` is None, that's totally legit, and the second line is identical. So you can just drop the `elif` check.

Reply:
ugh of course ><